test/ui/test_spec.js

```diff
@@ -88,7 +88,9 @@ describe('UI tests', function() {
               });
           }, 10000);
         })
-        .finally(() => done());
+        .then(driver.executeScript("window.datalab.vminfo.vm_name='fake_vm_name_for_testing';" +
+              "require(['static/appbar'],appbar=>appbar.setAccountMenuVminfo());"))
```

Comment:
Yeah, driver.execute returns a promise, so doesn't need to be wrapped in a function if that's the only call in the then arg.
This is already in the beforeAll block.
Yeah, I guess I could fix the sign-in button as well. It's a bit of a hack the way it is currently done, but at least it's the same on local and travis. :-)